### PR TITLE
Plat 2744 card io

### PR DESCRIPTION
**Pull request by bpwalford**

### CODE REVIEW

File: fitpay/src/main/java/com/fitpay/android/webview/enums/RtmType.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package com.fitpay.android.webview.enums;
+
+import android.support.annotation.StringDef;
+
+/**
+ * Rtm request & response types
+ */
+public class RtmType {
+
+    public static final int RTM_VERSION = 3;
```
Comment (ssteveli) [inline fitpay/src/main/java/com/fitpay/android/webview/enums/RtmType.java]: I don't remember seeing any V3 RTM code in the webapp, that's coming I assume?

File: fitpay/src/main/java/com/fitpay/android/webview/impl/WebViewCommunicatorImpl.java [hunk 2]
```diff
@@ -418,10 +434,16 @@ private void useDefaultParser(RtmMessage msg) {
...
                     break;
 
+                case RtmType.CARD_SCANNED:
+                    if (cardScanner != null) {
+                        cardScanner.startScan();
+                    }
+                    break;
+
                 default:
                     throw new IllegalArgumentException("unsupported action value in message with callbackId:" + callbackId);
```
Comment (ssteveli) [inline fitpay/src/main/java/com/fitpay/android/webview/impl/WebViewCommunicatorImpl.java]: Wanna just fix PLAT-2827 while you're at it?  This "seems" to be the place for it.

File: fitpay/src/main/java/com/fitpay/android/webview/enums/RtmType.java (hunk 1 above)
Comment (bpwalford) [inline fitpay/src/main/java/com/fitpay/android/webview/enums/RtmType.java]: I believe it's a high priority this sprint, but I'm not sure if Vlad intended this code to work with the current web view. I'll build the branch and make sure

**Comment by ssteveli** [comment]
Ya, it’ll be interesting to see how the web view reacts when an SDK moves ahead of it’s supported version.

Ultimately it should be a negotiation to the latest agreed upon version… so in this case the web app only supports 2, so the SDK should downgrade itself to 2.  Keeping them decoupled.

—Scott

File: fitpay/src/main/java/com/fitpay/android/webview/impl/WebViewCommunicatorImpl.java (hunk 2 above)
Comment (bpwalford) [inline fitpay/src/main/java/com/fitpay/android/webview/impl/WebViewCommunicatorImpl.java]: fixed

File: fitpay/src/main/java/com/fitpay/android/webview/enums/RtmType.java (hunk 1 above)
Comment (bpwalford) [inline fitpay/src/main/java/com/fitpay/android/webview/enums/RtmType.java]: Tested the web-view with this change and there doesn't appear to be any problems

Comment (skynet67) [inline fitpay/src/main/java/com/fitpay/android/webview/enums/RtmType.java]: RTM_VERSION represents the version that is supported by the SDK, but not the webview. If webview doesn't support V3 yet, then in the rtm message we'll receive "version"=2 and will use it.

**Pull request merged**